src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -101,7 +103,12 @@ public function flush()
         $this->data = array();
     }
 
-    public function push($stat, $value, $time = null)
+    /**
+     * @param string $stat
+     * @param string $value
+     * @param int|null $time
+     */
+    private function pushStat($stat, $value, $time = null)
```

Comment:
I still don't understand what is the benefit of renaming `push` to `pushStat`. IMHO, there is not benefit, and it breaks BC for nothing.